pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1120,8 +1116,8 @@ self: super: {
   # });
   libnix = dontCheck super.libnix;
 
-  # Jailbreak: https://github.com/jaor/xmobar/issues/463
-  # The test suite tries to mess with ALSA, which doesn't work in the build sandbox.
+  # > 0.33 => rm 464.patch
```

Comment:
In addition to a message like `> 0.33 => rm 464.patch`, would you be able to open a PR upstream asking for a new release?  (Or at least a Hackage revision...)  And then link to that issue here?

It is nice to have an upstream issue where we can potentially communicate with the upstream maintainer about releases.

Reply:
Ok.
I thought about it, obviously.
It is a good practice to the chain events.
I refuted it with that it is a responsibility of maintainers to release to fix builds, it is not our responsibility to push them to make releases.
Of course, we can foster them, and it would give us the link to check with the same util.

I thought of it just to basically have a CLI util that returns last Hackage version.

Also I straightly adopted a practice to cycle through open reports to new release. Made a release - close report, and open new report for the next release, have those default named reports, that seems logical both from an organizational and downstream community perspective.

I would open upstream report requests.

Reply:
Done.

This thing also can be automated.

Reply:
Ah, so this is what the Hackage revision system was made for.

Above, I asked for you to ask the maintainers to upload a new release, but in a lot of these cases, just a new revision of an existing release would be sufficient.  I would expect package maintainers to know this and be able to make the call as to whether to do a new release or just bump deps in a revision.

As to whether or not we should be reporting upstream, I feel strongly that we should.  Nixpkgs is often the very first place that versioning problems with various packages are found, since we automatically compile and distribute many packages from Hackage.  I think it is important that we give back to the Haskell community by reporting versioning constraints that are too tight, so they can be fixed quickly.

Although I agree that this could be automated better.  We would definitely welcome any tools to automate any part of the process of working with the Haskell stuff in nixpkgs!